### ISSUE TITLE
Uncaught TypeError: Cannot read property 'fromLatLngToDivPixel' of null

### ISSUE THREAD

**Issue by aleksandrlat** [JustFly1984/react-google-maps-api#1090]
# Issue template

If you want to ask question, ask it in [Spectrum.chat](https://spectrum.chat/react-google-maps) or [Slack channel](https://join.slack.com/t/react-google-maps-api/shared_invite/enQtODc5ODU1NTY5MzQ4LTBiNTYzZmY1YmVjYzJhZThkMGU0YzUwZjJkNGJmYjk4YjQyYjZhMDk2YThlZGEzNDc0M2RhNjBmMWE4ZTJiMjQ)

Please do not post unformatted code into issues, and please do not ask questions. Only real issues, PR or feature requests allowed. Minimal reproduction in codesandbox.io is must have.

## Please provide an explanation
This issue https://github.com/JustFly1984/react-google-maps-api/issues/875 was not fixed by https://github.com/JustFly1984/react-google-maps-api/pull/899 because we still see this error in our error logs with 1.9.7 version

## Your Environment

os: mac/linux/windows/android/ios

node --version

react version

webpack version

@babel version

@react-google-maps/api version: 1.9.7

## How does it behave?
We still see this error in our logs
```
File webpack:///./node_modules/@react-google-maps/api/dist/reactgooglemapsapi.esm.js line 2354 col 35 in [anonymous]
  var point = mapCanvasProjection.fromLatLngToDivPixel(position);
File webpack:///./node_modules/@react-google-maps/api/dist/reactgooglemapsapi.esm.js line 2372 col 156 in [anonymous]
  return bounds !== undefined ? getLayoutStylesByBounds(mapCanvasProjection, offset, ensureOfType...
File webpack:///./node_modules/@react-google-maps/api/dist/reactgooglemapsapi.esm.js line 2464 col 28 in 
  t.n.onPositionElement
var layoutStyles = getLayoutStyles(mapCanvasProjection, offset, _this.props.bounds, _this...
  File webpack:///./node_modules/@react-google-maps/api/dist/reactgooglemapsapi.esm.js line 2473 col 13 in Object.n.draw
_this.onPositionElement();
File webpack:///./node_modules/@react-google-maps/api/dist/reactgooglemapsapi.esm.js line 2512 col 71 in [anonymous]
  _this2.state.overlayView !== null && _this2.state.overlayView.draw();
```

## How should it behave correctly?
There should no be error

## basic implementation of incorrect behavior in codesandbox.com
I cannot reproduce myself

**Comment by Zazzzles** [comment]
Encountering same issue. Seems to be the same issue as discussed [here](https://github.com/google-map-react/google-map-react/issues/257)

**Comment by HFaifman** [comment]
I´ve only seen it on development and when using an overlayView as a custom marker. Any fix so far? Any ideas on how to reproduce it? It seems to be very random.

**Comment by HFaifman** [comment]
Shouldn´t this be relatively easy to fix though? Just make a check for mapCanvasProjection in dom helper. Trying to push my branch but it´s not letting me.

**Comment by JustFly1984** [comment]
released 1.9.8, please test

**Comment by aleksandrlat** [comment]
Looks like it helped. Here is small graphic of this error after we released new version to prod

![image](https://user-images.githubusercontent.com/3708309/88593525-589bc080-d068-11ea-843c-6537b2c9499b.png)

Thank you very much!
